  INFO root [0x00004950] [2018-04-29 15:56:12,245] - 
  INFO root [0x00004950] [2018-04-29 15:56:12,268] - -----------------------------------------------------------------------------------------


  INFO root [0x00004950] [2018-04-29 15:56:12,293] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x00004950] [2018-04-29 15:56:12,294] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x00004950] [2018-04-29 15:56:12,294] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x00004950] [2018-04-29 15:56:12,302] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:20015 )
  INFO root [0x00004950] [2018-04-29 15:56:12,302] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.106:20015, SOMAXCONN=128.
  INFO root [0x00004950] [2018-04-29 15:56:12,306] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:51797 )
  INFO root [0x00004950] [2018-04-29 15:56:12,306] - NetworkInterface::initialize(INTERNAL): address 192.168.0.106:51797, SOMAXCONN=128.
  INFO root [0x00004950] [2018-04-29 15:56:12,307] - server-configs:
  INFO root [0x00004950] [2018-04-29 15:56:12,307] - 	gameUpdateHertz : 10
  INFO root [0x00004950] [2018-04-29 15:56:12,307] - 	entryScriptFile : kbemain
  INFO root [0x00004950] [2018-04-29 15:56:12,307] - 	internalAddr : 192.168.0.106:51797
  INFO root [0x00004950] [2018-04-29 15:56:12,307] - 	externalAddr : 192.168.0.106:20015
  INFO root [0x00004950] [2018-04-29 15:56:12,307] - 	componentID : 7001
  INFO root [0x00004950] [2018-04-29 15:56:14,094] - ---- baseapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F9DFFB319FBADF35943ADF4D42C4718C. Config: Debug X86. Built: 15:53:31 Apr 29 2018. AppID: 7001. UID: 21461. PID: 15008 ----
  INFO root [0x00004950] [2018-04-29 15:56:14,094] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x00004950] [2018-04-29 15:56:14,095] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00004950] [2018-04-29 15:56:14,095] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00004950] [2018-04-29 15:56:14,218] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00004950] [2018-04-29 15:56:14,218] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00004950] [2018-04-29 15:56:14,218] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00004950] [2018-04-29 15:56:14,221] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00004950] [2018-04-29 15:56:14,222] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00004950] [2018-04-29 15:56:14,222] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00004950] [2018-04-29 15:56:14,222] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00004950] [2018-04-29 15:56:14,247] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00004950] [2018-04-29 15:56:14,303] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00004950] [2018-04-29 15:56:14,303] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00004950] [2018-04-29 15:56:14,432] - loaded entity-script:Account(1).
  INFO root [0x00004950] [2018-04-29 15:56:14,432] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00004950] [2018-04-29 15:56:14,432] - TelnetServer server is running on port 40000
  INFO root [0x00004950] [2018-04-29 15:56:14,712] - ---- baseapp is running ----
 DEBUG root [0x00004950] [2018-04-29 15:56:14,712] - Components::process(): Request for the process of identity...
  WARN root [0x00004950] [2018-04-29 15:56:20,818] - EntityApp::checkTickPeriod: tick took 6105% (6.11 seconds)!
  INFO root [0x00004950] [2018-04-29 15:56:21,895] - Components::findComponents: find logger(1)...
  INFO root [0x00004950] [2018-04-29 15:56:21,911] - Components::findComponents: found logger, addr:192.168.0.106:51799
  INFO root [0x00004950] [2018-04-29 15:56:21,911] - Components::addComponent: reset orderLog, uid=21461!
  INFO root [0x00004950] [2018-04-29 15:56:21,912] - Components::addComponent[logger], uid=21461, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00004950] [2018-04-29 15:56:21,913] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00004950] [2018-04-29 15:56:22,017] - Components::findComponents: found dbmgr, addr:192.168.0.106:51795
  INFO root [0x00004950] [2018-04-29 15:56:22,018] - Components::addComponent: reset orderLog, uid=21461!
  INFO root [0x00004950] [2018-04-29 15:56:22,018] - Components::addComponent[dbmgr], uid=21461, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00004950] [2018-04-29 15:56:22,019] - Components::findComponents: found dbmgr, addr:192.168.0.106:51795
  WARN root [0x00004950] [2018-04-29 15:56:23,019] - Forwarding logs to logger[192.168.0.106:51799/0/0/0]...
 ERROR root [0x00004950] [2018-04-29 15:56:34,324] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00004950] [2018-04-29 15:56:34,348] - The following logs sent to logger failed:
  WARN root [0x00004950] [2018-04-29 15:56:34,348] - ==> [2018-04-29 15:56:34 319] TCPPacketReceiver::processPendingEvents(192.168.0.106:51799): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00004950] [2018-04-29 15:56:34,348] - ==> [2018-04-29 15:56:34 324] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00004950] [2018-04-29 15:56:34,400] - TCPPacketReceiver::processPendingEvents(192.168.0.106:51795): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00004950] [2018-04-29 15:56:34,405] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00004950] [2018-04-29 15:56:34,428] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00004950] [2018-04-29 15:56:34,430] - onBaseAppShutDown: state=0
  INFO root [0x00004950] [2018-04-29 15:56:34,430] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00004950] [2018-04-29 15:56:34,430] - onBaseAppShutDown: state=1
  INFO root [0x00004950] [2018-04-29 15:56:34,430] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00004950] [2018-04-29 15:56:34,431] - onBaseAppShutDown: state=2
 DEBUG root [0x00004950] [2018-04-29 15:56:34,439] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00004950] [2018-04-29 15:56:34,439] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00004950] [2018-04-29 15:56:34,439] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00004950] [2018-04-29 15:56:34,439] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00004950] [2018-04-29 15:56:34,439] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00004950] [2018-04-29 15:56:34,439] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00004950] [2018-04-29 15:56:34,439] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00004950] [2018-04-29 15:56:34,440] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00004950] [2018-04-29 15:56:34,449] - Script::uninstall(): is successfully!
 DEBUG root [0x00004950] [2018-04-29 15:56:34,449] - ThreadPool::destroy(): starting size 2.
